module1.js

```diff
@@ -53,6 +59,14 @@ function question3(number) {
 */
 
 function question4(array) {
+  for (i = 0; i < array.length; i++) {
```

Comment:
A lot going on here. Can you take a second try here? What were you confused about when working through this problem?